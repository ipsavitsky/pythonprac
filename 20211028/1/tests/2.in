print(', '.join([str(i) for i in fib(40, 50)]))