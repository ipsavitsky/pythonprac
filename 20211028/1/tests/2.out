165580141
267914296
433494437
701408733
1134903170
1836311903
2971215073
4807526976
7778742049
12586269025
20365011074
